policy/Object.yaml

```diff
@@ -0,0 +1,392 @@
+funcs:
+    objectId: # the index of deepCoadd_obj IS the objectId
+        functor: Index
+    parentObjectId:
+        functor: Column
+        args: parent
+        dataset: ref
+    coord_ra:
+        # reference position for merged "ref" cat. Required by db. Not in DPDD
+        functor: RAColumn
```

Comment:
Should the “A” of RA really be capitalized?

Also, there’s a note about why coord_dec below, but no accompanying one for coord_ra here.  Deliberate?